code/builders/builder-vite/src/virtual-file-names.ts

```diff
@@ -1,4 +1,4 @@
-export const virtualFileId = '/virtual:/@storybook/builder-vite/vite-app.js';
```

Comment:
@IanVS Is the leading forward slash required?

Reply:
Not as far as I know.